src/block_service/fds/fds_service.cpp

```diff
@@ -37,9 +37,11 @@ DSN_DEFINE_uint64("pegasus.server",
                   64 << 20,
                   "rocksdb options.write_buffer_size");
 
-DSN_DEFINE_uint32("replication", fds_write_limit_rate, 100, "rate limit of fds(MB/s)");
+DSN_DEFINE_uint32("replication", fds_write_limit_rate, 100, "write rate limit of fds(MB/s)");
```

Comment:
I recommend those three configuration can be updated dynamically

Reply:
Do you mean add a remote command? I think we can add it in next pull request.

Reply:
Yes~ I will expect your further pull request.